src/win/winapi.c

```diff
@@ -63,6 +64,10 @@ void uv_winapi_init(void) {
     uv_fatal_error(GetLastError(), "GetModuleHandleA");
   }
 
+  pRtlGetVersion = (sRtlGetVersion) GetProcAddress(
```

Comment:
How about instead of doing this version check, you just call CreateSymbolicLink with the flag, and if you get back `ERROR_INVALID_PARAMETER`, you do it again without the flag.